helm_deploy/demo-app/templates/ingress.yaml

```diff
@@ -0,0 +1,18 @@
+
+apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
+  name: django-ingress
+  annotations:
+    kubernetes.io/ingress.class: "nginx"
+    kubernetes.io/tls-acme: "true"
```

Comment:
We don't need this annotation anymore - it's for Let's Encrypt, which we've dropped.